dist/lovefield.js

```diff
@@ -1,6 +1,6 @@
-if(!self.window){window=this;}
+if(!self.window){window=self;}
 (function(){'use strict';var $jscomp = {scope:{}}, goog = goog || {};
-goog.global = this;
+goog.global = self;
```

Comment:
Is this line change necessary for the fix? This one we can't really affect by changing the export script, because this line comes straight from Closure library.